lib/OpenQA/WebAPI/Plugin/ObsRsync/Controller/Gru.pm

```diff
@@ -128,7 +128,8 @@ sub run {
     return $self->render(json => {message => 'queue full'}, status => QUEUE_FULL)
       if ($results->{total} >= $queue_limit);
 
-    $app->gru->enqueue('obs_rsync_run', {project => $project, url => $helper->project_status_url}, {priority => 100});
+    my $url = $helper->get_api_dirty_status_url($project);
+    $app->gru->enqueue('obs_rsync_run', {project => $project, url => $url}, {priority => 100});
```

Comment:
Suggestion from the call: Try and check the minion dashboard contents based on how it's done in t/ui/27-plugin_obs_rsync_gru.t